### PR TITLE
Optimize invoice find function

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR optimize the invoice find function by forcing to use index on patient table

### CODE REVIEW

File: server/controllers/finance/patientInvoice.js [hunk 1]
```diff
@@ -234,8 +234,7 @@ function find(options) {
       em.text AS patientReference, service.name as serviceName, proj.name AS project_name,
       user.display_name, invoice.user_id, invoice.reversed, invoice.edited
     FROM invoice
-    LEFT JOIN patient ON invoice.debtor_uuid = patient.debtor_uuid
```
Comment (jniles) [inline server/controllers/finance/patientInvoice.js:237]: I've never seen FORCE INDEX before.  From what I am reading, this looks like the right solution!

**Review by jniles — APPROVED** [review]
@mbayopanda does this speed up the query on your machine?  How long does it take now?

File: server/controllers/finance/patientInvoice.js (hunk 1 above)
Comment (mbayopanda) [reply server/controllers/finance/patientInvoice.js:237]: @jniles it's now very fast... 460ms instead of 11seconds

**Comment by jniles** [comment]
bors r+

Let's try it then!

**Pull request merged**